api/envoy/api/v2/listener/listener_components.proto

```diff
@@ -255,17 +254,14 @@ message ListenerFilter {
     google.protobuf.Any typed_config = 3;
   }
 
-  // [#not-implemented-hide:]
   // Decide when to disable this listener filter on incoming traffic.
-  // Example:
-  // 0. always enable filter
-  //    don't set `filter_disabled`
-  // 1. disable when the destination port is 3306
-  //    rule.destination_port_range = Int32Range {start = 3306, end = 3307}
-  // 2. disable when the destination port is 3306 or 15000
-  //    rule.or_match = MatchSet.rules [
-  //       rule.destination_port_range = Int32Range {start = 3306, end = 3307},
-  //       rule.destination_port_range = Int32Range {start = 15000, end = 15001},
-  //    ]
+  //
+  // Examples:
+  //
+  // * clear the field if filter is always enabled
+  // * disable when the destination port is 3306
+  //   ``rule.destination_port_range = Int32Range {start = 3306, end = 3307}``
+  // * disable when the destination port is 3306 or 15000
```

Comment:
Random drive-by, looks like this is a bit of a regression in API docs, they've lost some examples.

Reply:
Thanks for the comment! I made this change because the # of lines of the comments is large IMHO. 
I feel the doc should describe the default behavior of the field and leave the examples to the message type doc.

I can revert if you prefer the verbose version before this PR

Reply:
I think either get rid of the example and point to a full one in the docs or just provide some self-contained examples here. Having half formed examples isn't that great.

Reply:
+1 to doing the full example in one place or the other. Please also add a release note in addition to any arch level docs.